spec/octokit/client/reviews_spec.rb

```diff
@@ -127,6 +127,18 @@
             assert_requested :get, requested_url
           end
         end
+
+        describe '.delete_request_pull_request_review' do
+          it 'deletes a requests for a pull request review' do
```

Comment:
The test needs to use VCR in order for the specs to pass on CI.

Reply:
@kytrinyx I know I have to create a VCR cassette but I am having troubles with it. I have no idea how can I record a new cassette and I have `401 - Bad credentials` problem at recording.

Reply:
I don't think there is a guide—we definitely need one (we run into this a lot).

Do you have a valid personal token with the right scopes to make this request?
If you do, are you setting the `OCTOKIT_TEST_GITHUB_TOKEN` env var?

Reply:
Are you able to use the token to make a request using curl?

If there is an existing VCR cassette, can you delete it and try again? (I can't remember if you said that you aren't getting a cassette at all).

Reply:
I am able to perform any request successfully via `curl` with a valid token.

Create repository example:
```sh
curl -H "Authorization: token $OCTOKIT_TEST_GITHUB_TOKEN" https://api.github.com/user/repos -d '{"name": "test_repository"}'
```
```json
{
  "id": 126463333,
  "name": "test_repository",
  "full_name": "europ/test_repository",
  "owner": {
    "login": "europ",
    "id": 22373707,
...
```

I am having troubles with a new VCR cassette creation for `delete_request_pull_request_review`. After the recording, it results in a failure (described above - bad credentials) and in a creation of a new cassette
```
spec/cassettes/Octokit_Client_Reviews/with_repository/with_pull_request/with_pull_request_review_request/_delete_pull_request_review_request/deletes_a_requests_for_a_pull_request_review.json
```
which has incorrect content of `"body"` section - it contains a few data in comparison to other cassettes which have a huge amount of data in this section. I have deleted this `deletes_a_requests_for_a_pull_request_review.json` cassette and tried again but without any success.

Can you, please, give me a step by step instruction guide how should I create a corresponding VCR cassette for my added [test](https://github.com/octokit/octokit.rb/pull/990/files#diff-8f4c6f7c6161090a4cd5e46053a55d68R132)?

Reply:
I am able to perform any request successfully via `curl` with a valid accessible token.

Example (create repository):
```sh
curl -H "Authorization: token $OCTOKIT_TEST_GITHUB_TOKEN" https://api.github.com/user/repos -d '{"name": "test_repository"}'
```
Result (created [repository](https://github.com/europbot/test_repository)):
```json
{
  "id": 126479726,
  "name": "test_repository",
  "full_name": "europbot/test_repository",
  "owner": {
    "login": "europbot",
    "id": 36951445,
...
```

I am having troubles with a new VCR cassette creation for `delete_request_pull_request_review`.

---

**The recording results in a failure and in a creation of a new cassette which is inapplicable**.

I have set the required environment variables as it was desired. I have created my repository via curl. If I would not create the repository it would result in a failure of bad credentials described before: `Failure/Error: @repo = @client.create_repository('test-repo', auto_init: true)`.

Used commands in the following order:
```sh
export OCTOKIT_TEST_GITHUB_LOGIN=europbot
export OCTOKIT_TEST_GITHUB_TOKEN=12345678
rm -rf spec/cassettes/Octokit_Client_Reviews/with_repository/with_pull_request/with_pull_request_review_request/_delete_pull_request_review_request/
curl -H "Authorization: token $OCTOKIT_TEST_GITHUB_TOKEN" https://api.github.com/user/repos -d '{"name": "test_repository"}'
bundle exec rspec spec/octokit/client/reviews_spec.rb
```

### Result:
* **failure**:

```                                                                          
....F......

Failures:

  1) Octokit::Client::Reviews with repository with pull request with pull request review request .delete_pull_request_review_request deletes a requests for a pull request review
     Failure/Error: @client.request_pull_request_review(@repo.full_name, @pull.number,
     Octokit::UnprocessableEntity:
       POST https://api.github.com/repos/europbot/test-repo/pulls/1/requested_reviewers: 422 - Reviews may only be requested from collaborators. One or more of the users or teams you specified is not a collaborator of the europbot/test-repo repository. // See: https://developer.github.com/v3/pulls/review_requests/#create-a-review-request
     # ./lib/octokit/response/raise_error.rb:16:in `on_complete'
     # ./lib/octokit/middleware/follow_redirects.rb:73:in `perform_with_redirection'
     # ./lib/octokit/middleware/follow_redirects.rb:61:in `call'
     # ./lib/octokit/connection.rb:156:in `request'
     # ./lib/octokit/connection.rb:28:in `post'
     # ./lib/octokit/client/reviews.rb:161:in `request_pull_request_review'
     # ./spec/octokit/client/reviews_spec.rb:107:in `block (5 levels) in <top (required)>'

Finished in 8.1 seconds (files took 0.31834 seconds to load)
11 examples, 1 failure

Failed examples:

rspec ./spec/octokit/client/reviews_spec.rb:132 # Octokit::Client::Reviews with repository with pull request with pull request review request .delete_pull_request_review_request deletes a requests for a pull request review
```

* **new created cassette**:
```
spec/cassettes/Octokit_Client_Reviews/with_repository/with_pull_request/with_pull_request_review_request/_delete_pull_request_review_request/deletes_a_requests_for_a_pull_request_review.json
```

The main problem is at this [line](https://github.com/europ/octokit.rb/blob/61aed21414ba443fa3581cdb1aeb51e0353c29b5/spec/octokit/client/reviews_spec.rb#L101) in test. The collaborators cannot be requested for any action (e.g. review request) until they do not confirm the repository invitation. I have added them manually but I am unable to accept this invitation for them. What should I do?

Can you, please, give me a step by step instruction guide how should I create a corresponding VCR cassette for my added [test](https://github.com/octokit/octokit.rb/pull/990/files#diff-8f4c6f7c6161090a4cd5e46053a55d68R132)?

Reply:
>  until they do not confirm the repository invitation.

Aaah! Yes. That makes total sense, and I see the problem. Let me think about the best way forward (it might be for me or @tarebyte to generate the cassette). I'm sorry this is so painful! It's not supposed to be that way.

Reply:
I am glad to hear that :blush:. Please, let me know how to proceed with the completion.